tests/test_packages/test_connections/test_p2p_libp2p/test_fault_tolerance.py

```diff
@@ -78,85 +72,69 @@ def change_state_and_wait(
             lambda: multiplexer.is_connected == expected_is_connected, timeout=timeout
         )
 
-    def teardown(self):
-        """Tear down the test"""
-        for mux in self.multiplexers:
-            mux.disconnect()
-        os.chdir(self.cwd)
-        try:
-            shutil.rmtree(self.t)
-        except (OSError, IOError):
-            pass
-
 
 @pytest.mark.skip
 @libp2p_log_on_failure_all
 class TestLibp2pConnectionRelayNodeRestartIncomingEnvelopes(BaseTestLibp2pRelay):
     """Test that connection will reliably receive envelopes after its relay node restarted"""
 
-    @libp2p_log_on_failure
-    def setup(self):
+    @classmethod
+    def setup_class(cls):
```

Comment:
this was using setup instead of the setup_class classmethod, and was therefore inconsistent with the rest